app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -20,12 +21,18 @@ data class Event(
 ) {
   constructor(
       map: Map<String, Any>,
-      id: String
+      id: String,
   ) : this(
       eventName = map["name"] as String,
       eventPhoto = map["photo_url"] as String,
-      start = LocalDateTime.parse(map["start"] as String),
-      end = LocalDateTime.parse(map["end"] as String),
+      start =
+          LocalDateTime.parse(
+              map["start"] as String, DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm"))
+              as LocalDateTime,
+      end =
+          LocalDateTime.parse(
+              map["end"] as String, DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm"))
+              as LocalDateTime,
```

Comment:
this currently crashes the app. but what is the point of this? it seems to work well already.

Reply:
It was to ensure that upon adding an event to firebase database, that it would always have a DateTime pattern of "yyyy-MM-dd'T'HH:mm". I had some errors with DateTime parsing while I was writing the tests so making it the strict pattern helped me find other sources of error.

There are 2 possible fixes to this:
Using the ISO_Local_Date_Time format (which is basically the `"yyyy-MM-dd'T'HH:mm"` format)
or 
Not providing a format, and just parsing it into a date time.

I'll keep it the second one for now, to potentially account for different date formats in different regions.